cmd/k8s-operator/connector.go

```diff
@@ -244,21 +252,28 @@ func (a *ConnectorReconciler) maybeProvisionConnector(ctx context.Context, logge
 		return err
 	}
 
-	dev, err := a.ssr.DeviceInfo(ctx, crl, logger)
+	devices, err := a.ssr.DeviceInfo(ctx, crl, logger)
 	if err != nil {
 		return err
 	}
 
-	if dev == nil || dev.hostname == "" {
-		logger.Debugf("no Tailscale hostname known yet, waiting for Connector Pod to finish auth")
-		// No hostname yet. Wait for the connector pod to auth.
-		cn.Status.TailnetIPs = nil
-		cn.Status.Hostname = ""
-		return nil
+	hostnames := make([]string, 0)
+	cn.Status.TailnetIPs = make([]string, 0)
+
+	for _, dev := range devices {
+		if dev == nil || dev.hostname == "" {
+			continue
+		}
+
+		for _, ip := range dev.ips {
+			cn.Status.TailnetIPs = append(cn.Status.TailnetIPs, ip)
+		}
+
+		// TODO(davidsbond): we likely don't want to override this every time.
+		hostnames = append(hostnames, dev.hostname)
 	}
 
-	cn.Status.TailnetIPs = dev.ips
-	cn.Status.Hostname = dev.hostname
+	cn.Status.Hostname = strings.Join(hostnames, ",")
```

Comment:
Would it make sense to change the CRD so that it is an array field? Rather than one string field with comma separation for each hostname.